src/Catalog/Helpers/FeedPackageIdentity.cs

```diff
@@ -30,12 +30,12 @@ public FeedPackageIdentity(PackageIdentity package)
 
         public bool Equals(FeedPackageIdentity other)
         {
-            return Id == other.Id && Version == other.Version;
+            return Id.ToLowerInvariant() == other.Id.ToLowerInvariant() && Version.ToLowerInvariant() == other.Version.ToLowerInvariant();
```

Comment:
please add UTs that make sure Equals and GetHashCode are equivalent. I.e. if Equals returns true for two FeePackageIdentity classes, their hash codes are also identical (and vice versa).